cmd/bad-key-revoker/main.go

```diff
@@ -76,6 +82,19 @@ func (ubk uncheckedBlockedKey) String() string {
 		ubk.RevokedBy, ubk.KeyHash)
 }
 
+func (bkr *badKeyRevoker) countUncheckedKeys() (int, error) {
+	var count int
+	err := bkr.dbMap.SelectOne(
+		&count,
+		`SELECT COUNT(*)
+		FROM (SELECT * FROM blockedKeys
```

Comment:
```suggestion
		FROM (SELECT 1 FROM blockedKeys
```

It's possible MariaDB could do this optimization for us but we might as well tell it it does not need to read the actual columns from each row.